src/exchange/exchange_client.rs

```diff
@@ -172,6 +175,56 @@ impl ExchangeClient {
         self.post(action, signature, timestamp).await
     }
 
+    pub async fn class_transfer(
+        &self,
+        usdc: f64,
+        to_perp: bool,
+        wallet: Option<&LocalWallet>,
+    ) -> Result<ExchangeResponseStatus> {
+        let usdc = (usdc * 1e6).round() as u64;
```

Comment:
nit: add comment for what this means, i.e. payload expects usdc without decimals

Reply:
Done